features/FxA-28-change-email/README.md

```diff
@@ -0,0 +1,53 @@
+
+Change Email
+======================
+
+https://mozilla.aha.io/features/FXA-28
+
+As an FxA user when I lose control of the email account  that is my username in Firefox Accounts I want to be able to change that FxA email address so that I don't lose access to my data and other FxA-relying services.
+* When a user changes their email in Firefox Accounts, it simply updates in other clients and relying services like display name without requiring them to reconnect.
```

Comment:
What **Display name** means is that when I update my display name on the web, it just changes in the client (e.g. hamburger menu) without any fuss. There no need for an explicit notification to the user in the client. I think all the confirmation the user needs is provided in FxA settings after the verify the new email.